messenger/message-recorder.rst

```diff
@@ -33,88 +33,14 @@ Doctrine transaction, in which the user has been created.
-transaction will not be rolled back.
-
-The  ``dispatch_after_current_bus`` middleware is enabled by default. It is configured as the
-first middleware on all busses. When doing a highly custom or special configuration, then make
-sure ``dispatch_after_current_bus`` is registered before ``doctrine_transaction``
-in the middleware chain.
-
-**Note:** The ``dispatch_after_current_bus`` middleware must be loaded for *all* of the
-buses. For the example, the middleware must be loaded for both the command and event bus.
-
-.. configuration-block::
-
-    .. code-block:: yaml
-
-        # config/packages/messenger.yaml
-        framework:
-            messenger:
-                default_bus: messenger.bus.command
-
-                buses:
-                    messenger.bus.command:
-                        middleware:
-                            - validation
-                    messenger.bus.event:
-                        default_middleware: allow_no_handlers
-                        middleware:
-                            - validation
-
-    .. code-block:: xml
-
-        <!-- config/packages/messenger.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:framework="http://symfony.com/schema/dic/symfony"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                https://symfony.com/schema/dic/services/services-1.0.xsd">
-
-            <framework:config>
-                <framework:messenger default_bus="messenger.bus.command">
-                  <framework:bus name="messenger.bus.command">
-                      <framework:middleware id="validation">
-                      <framework:middleware id="doctrine_transaction">
-                  </framework:bus>
-                  <framework:bus name="messenger.bus.command" default_middleware="allow_no_handlers">
-                      <framework:middleware id="validation">
-                      <framework:middleware id="doctrine_transaction">
-                  </framework:bus>
-                </framework:messenger>
-            </framework:config>
-        </container>
-
-    .. code-block:: php
-
-        // config/packages/messenger.php
-        $container->loadFromExtension('framework', [
-            'messenger' => [
-                'default_bus' => 'messenger.bus.command',
-                'buses' => [
-                    'messenger.bus.command' => [
-                        'middleware' => ['validation', 'doctrine_transaction'],
-                    ],
-                    'messenger.bus.event' => [
-                        'default_middleware' => 'allow_no_handlers',
-                        'middleware' => ['validation', 'doctrine_transaction'],
-                    ],
-                ],
-            ],
-        ]);
-
```

Comment:
This code block didn't seem relevant anymore. I think it originally came from when `dispatch_after_current_bus` was not going to be included as a default middleware, so you needed to show how it was added to config.